nuitka/plugins/standard/ImplicitImports.py

```diff
@@ -97,7 +97,9 @@ def _getImportsByFullname(self, module, full_name):
         if config:
             for entry in config:
                 if entry.get("control_tags"):
-                    if not self.evaluateControlTags(entry.get("control_tags")):
+                    if not self.evaluateControlTags(
+                        full_name, entry.get("control_tags")
```

Comment:
Once it's more than one argument, I prefer to pass as keyword arguments for clarity, but it's not strict requirement.

As aside, once we might want to put the module name also into context, because I imagine at some point, we will define rules, that work on pattern matches rather than exact module names, in which case, the condition could still be applied, so passing this is not only good for the error message and information keeping, but also for the actual conditions mid term. But now of course, doesn't make yet sense to add it.